client/post-editor/post-editor.jsx

```diff
@@ -343,11 +323,26 @@ const PostEditor = React.createClass( {
 								site={ site }
 								type={ this.props.type }
 							/>
+							<div className="editor__site">
```

Comment:
The main component in that bar is the site, the status is like an small add-on. Also, I'm trying to rationalize my decision because I can't come up with a better name, if you do I'll search&replace it :)